interview-questions.md

```diff
@@ -8,33 +8,33 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: Calling a super causes a class to inherit all of its properties, without having to initialize it again.
 
-Researched answer:
+Researched answer: Calling super causes the Superclass to pass down information to the subclass. This interaction is called inheritance, and is used by the "<" character. A subclass will inherit all of the data that the Superclass provides. The super must also be invoked in the initialization, before instance variables are declared. 
 
 2. What is a gem?
 
-Your answer:
+Your answer: A gem is a ruby dependency file. We use it mainly for testing. There are many ruby gems. 
 
-Researched answer:
+Researched answer: A gem is a snippet of code functionality that is managed by a packet manager. It allows you to download, and use ruby software packages on your system. They add large amounts of functionality to your software.
 
 3. What is a relational database? Are there other kinds of databases?
 
 Your answer:
```

Comment:
I think you missed this question, can you please go back and complete this so I can review it

Reply:
Hey Nicole, I went back and completed this answer, thanks.